### DISCUSSION TITLE
nginx.conf.sigil since index out of range

### DISCUSSION THREAD

**Discussion by jaredmoody** [dokku/dokku#5159] [Q&A]
Hi, just trying to get started with dokku for the first time and got close to making it though the tutorial in the docs when I hit this error on push:

`template: nginx.conf.sigil:4:19: executing "nginx.conf.sigil" at <index $port_map_list 1>: error calling index: reflect: slice index out of range`

I thought maybe I had missed mapping the port, so I tried this with the same error:

```
$ dokku proxy:ports-add my-app http:80:8080
-----> Setting config vars
       DOKKU_PROXY_PORT_MAP:  http:80:8080
-----> No proxy port set, setting to random open high port
template: nginx.conf.sigil:4:19: executing "nginx.conf.sigil" at <index $port_map_list 1>: error calling index: reflect: slice index out of range
2022/05/06 05:15:55 exit status 1
```

I'm perplexed that it says "No proxy port set" when I'm trying to define one.

Can someone steer me in the right direction here?

Thanks!

**Comment by Sereton** [comment]
I have the same problem. Using a fresh installation of Ubuntu 20.04 and a working ROR app

**Comment by josegonzalez** [comment]
This sounds like a bug. Mind filing an issue with all the requested data so it can be investigated?

**Comment by josegonzalez** [comment]
If you're having this issue, see the fix in #5190. The permissions on the ENV file for your apps may be incorrect.
